allensdk/api/queries/cell_types_api.py

```diff
@@ -27,15 +27,19 @@ class CellTypesApi(RmaApi):
     def __init__(self, base_uri=None):
         super(CellTypesApi, self).__init__(base_uri)
 
-    def list_cells(self,
-                   require_morphology=False,
-                   require_reconstruction=False,
+    def list_cells(self, 
+                   id=None, 
```

Comment:
Can this be a list of `ids`, instead of just one?  It would match what we've done in BrainObservatoryApi a bit better.